include/core/utilities.h

```diff
@@ -591,21 +593,21 @@ value_string_to_tensor(const std::string &value_string_0,
  *
  * @tparam dim Number of spatial dimensions (2D or 3D).
  *
- * @param[in] cell_measure Area (2D) or volume (3D) of the cell.
+ * @param[in] cell_volume Area (2D) or volume (3D) of the cell.
  *
  * @param[in] fe_degree Polynomial degree of the shape function.
  *
  * @return Cell diameter value.
  */
 template <int dim>
 inline double
-compute_cell_diameter(const double cell_measure, const unsigned int fe_degree)
+compute_cell_diameter(const double cell_volume, const unsigned int fe_degree)
```

Comment:
the variable cell_volume is not a very good choice. I prefer the word measure because the measure is the area or the volume depending on dim